pytest/tests/shardnet/README.md

```diff
@@ -1,17 +0,0 @@
-# Shardnet tools
```

Comment:
uh - these tools seem to be quite generic. I'm surprised that they are in "pytest" directory (As they are not really tests).

Can you check with @nikurt / @marcelo-gonzalez  - to see which ones are still useful.

Reply:
Please keep all of them